### ISSUE TITLE
OpenSSL install on user machine?

### ISSUE THREAD

**Issue by JelleBakker** [paullouisageneau/libdatachannel#788]
I'm trying to deploy an app on Windows and Mac using OpenSSL.
I'm struggling to understand how this must be done considering the libdatachannel.
When I try to run the compiled app on a mac machine I get a crash with following message:
```
Termination Reason:    Namespace DYLD, Code 1 Library missing
Library not loaded: '/usr/local/opt/openssl@1.1/lib/libcrypto.1.1.dylib'
```

Does this mean I should install OpenSSL on the users machine as part of my app installation?
Or is it possible to statically link with OpenSSL when compiling so I don't have a dynamic dependency.

Looking forward to hearing your thoughts...

Jelle

**Comment by paullouisageneau** [comment]
>  Or is it possible to statically link with OpenSSL when compiling so I don't have a dynamic dependency.

Yes, you indeed need to build OpenSSL as a static library and link against it.
